guides/doc-Managing_Configurations_Puppet/docinfo.xml

```diff
@@ -0,0 +1,14 @@
+<title>Managing Configurations Using Puppet Integration in Satellite</title>
+<productname>Red Hat Satellite</productname>
+<productnumber>7.0</productnumber>
+<subtitle>Using Puppet integration to manage configurations of hosts with Puppet modules and reporting to Satellite</subtitle>
+<abstract><para>This guide describes how to enable Puppet integration with Satellite, configure the Puppet agent on hosts, how to import Puppet modules, and how to use the Puppet modules to enforce configurations on hosts managed in your Red&nbsp;Hat Satellite infrastructure.</para></abstract>
```

Comment:
Pretty good overall, but one thing I'd change is the self-referential wording at the start. Our editor in RH always highlights that we should avoid self references like 'this chapter discusses <x>' or 'this document details <x>' because it removes the reader from their task and bring the writer talking about their guide into the equation.
So I'd change it to something more user action oriented like:

In Red&nbsp;Hat Satellite, you can enable Puppet integration, configure the Puppet agent on hosts, import Puppet modules, and use the Puppet modules to enforce configurations on hosts managed in your Red&nbsp;Hat Satellite infrastructure.

Reply:
Okay, just for the record: We've decided to keep this for now, so the wording is consistent with the other guides on RH Customer Portal. Noting for future improvements.